content/en/docs/setup/install/istioctl/index.md

```diff
@@ -303,345 +303,6 @@ Then run the following `verify-install` command to see if the installation was s
 $ istioctl verify-install -f $HOME/generated-manifest.yaml
 {{< /text >}}
 
-## Customizing the configuration
```

Comment:
can we have a link to the other doc here? otherwise most will not find it

Reply:
Added. Thanks.